code/datums/uplink_item.dm

```diff
@@ -480,7 +487,7 @@ var/list/uplink_items = list()
 	name = "Stimpack"
 	desc = "Stimpacks, a tool of many great heroes, make you nearly immune to stuns and knockdowns for about 5 minutes after injection."
 	item = /obj/item/weapon/reagent_containers/syringe/stimulants
-	cost = 8
+	cost = 5 //it's much, much worse than an adrenal implant

```

Comment:
I've used the stimpack - it doesn't seem to instantly pop you back up like the adrenal does, even if it's a duration rather than a series of jolts. Additionally, you can pop an adrenal implant at the start of the round, then use it when security start closing in, then once more when you get slipped on the shuttle and a mime approaches with with a stunprod. A stimpack can't do that. It's more useful, for, say, a nuke op assaulting the brig than a traitor trying to not get asimov-caged or whatever.